### DISCUSSION TITLE
Training with multiple DFT levels

### DISCUSSION THREAD

**Discussion by ald3hyde** [deepmodeling/deepmd-kit#2938] [Q&A]
Dear all.
I am currently trying to create a model using data calculated at multiple DFT levels, the main difference between the DFT levels is the basis, I would like to combine the plane wave basis used in VASP with the atomic basis used in CP2K, etc. How is this feasible?
If I composite them directly, the RMSE error will be too large.

**Comment by njzjz** [comment]
You may want to train multiple models, but they share the same descriptor and some fitting layers. See https://docs.deepmodeling.com/projects/deepmd/en/master/train/multi-task-training.html. Our software paper also describes it.
